harvard-university-for-the-creative-arts.csl

```diff
@@ -82,9 +90,22 @@
             <group delimiter=" ">
               <group delimiter=" ">
                 <text variable="title" font-style="italic" suffix="."/>
+                <choose>
+                  <if match="any" variable="volume">
+                    <text term="volume" form="short" text-case="capitalize-first"/>
+                    <text variable="volume" text-case="uppercase" suffix="."/>
+                  </if>
+                </choose>
                 <text macro="edition-no"/>
+                <group>
+                  <choose>
+                    <if match="any" variable="collection-title">
+                      <text term="in" text-case="capitalize-first" suffix=": "/>
+                      <text variable="collection-title" font-style="normal" suffix="."/>
+                    </if>
+                  </choose>
+                </group>
```

Comment:
I understand now, thanks for that. I mainly edit on the visual editor. I need to get in the habit of looking at the differences I make to the code view when I edit something.

Thanks again